tests/test_project.py

```diff
@@ -2485,7 +2485,7 @@ def test_project_schema_versions(self):
         assert version.parse(self.project.config["schema_version"]) < version.parse(
             impossibly_high_schema_version
         )
-        config = get_config(self.project.fn("signac.rc"))
+        config = _get_config(self.project.fn("signac.rc"))
```

Comment:
Can this test (and others below) be rewritten to use the functions that will be in the public API in 2.0?